### DISCUSSION TITLE
Are there any signs of life on this project?

### DISCUSSION THREAD

**Discussion by agrrobert** [Serial-Studio/Serial-Studio#153] [General]
Issues regarding Dashboard being greyed out on Windows 10 have been open since December.

**Comment by alex-spataru** [comment]
There are signs of life! I've just released a new version with a host of exciting features and improvements.

That said, I want to mention that managing a large open-source project can be challenging, especially when there's limited support from the user community. As a developer, I genuinely appreciate user feedback, but it can be draining to constantly fix issues without much support in return.

If you'd like to help keep the project alive, I encourage you (and anyone reading this) to support it in any way you can—whether through [donations](https://www.paypal.com/donate?hosted_button_id=XN68J47QJKYDE), contributing to the [wiki](https://github.com/Serial-Studio/Serial-Studio/wiki), creating [examples](https://github.com/Serial-Studio/Serial-Studio/tree/master/examples), or even writing guides or making pull requests.

Please note that I am not obligated to provide free technical support or fix every issue, but I do my best when I have the time and energy. Users who expect free fixes and priority support without contributing may find [this talk](https://www.youtube.com/watch?v=vzYqxo13I1U&t=2646s) enlightening. I am currently updating Serial Studio because it directly benefits my clients and ongoing projects, not because I enjoy working for free or have nothing better to do.

And another thing: creating an issue with nothing more than "It doesn't work," with no other information or effort to research the problem, will get you ignored. In some cases, I may even close the issue for lack of information that allows me to replicate the problem. Please provide details and context if you want to get meaningful help 😄
